app/config/config_dev.yml

```diff
@@ -23,6 +23,16 @@ monolog:
 #        chromephp:
 #            type:  chromephp
 #            level: info
+        console:
+          type: console
+          process_psr_3_messages: false
+          channels: ['!event', '!doctrine', '!console']
+        # To follow logs in real time, execute the following command:
```

Comment:
Ok, it seems fair. I didn't have a look at the other config files. It just seamed more clear just under `console`.
Fixed 49abb52.